src/main/java/pl/javastart/task/Array2dController.java

```diff
@@ -0,0 +1,64 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class Array2dController {
+    private final int maxColAmount = 100;
+    private final int maxRowAmount = 100;
+
+    private int[][] array;
+
+    public Array2dController() {
+        Scanner scanner = new Scanner(System.in);
+        int rowAmount = getRowAmount(scanner);
+        int colAmount = getColumnAmount(scanner);
+        array = new int[rowAmount][colAmount];
+        fillArray();
+        scanner.close();
+    }
```

Comment:
Unikaj tak rozbudowanej inicjalizacji w ramach konstruktora. Technicznie nie jest to błąd, ale później w metodzie main z zapisu:
```
Array2dController controller = new Array2dController();
controller.showArray();
```
tak naprawdę zupełnie nie wynika jak wiele rzeczy dzieje się pod spodem.